filebeat/docs/inputs/input-filestream-reader-options.asciidoc

```diff
@@ -141,3 +141,92 @@ The default is 16384.
 
 The maximum number of bytes that a single log message can have. All bytes after
 `mesage_max_bytes` are discarded and not sent. The default is 10MB (10485760).
+
+[float]
+===== `parsers`
+
+This option expects a list of parsers the log line has to go through. 
+
+Avaliable parsers:
+- `multiline`
+- `ndjson`
+
+In this example a JSON object that spans 3 lines are aggregated into a single
+event and then parsed by `ndjson`.
```

Comment:
Maybe want want to invert the example. Parse the JSON and do multiline on the JSON contents.